pkg/anchor/activity/activity.go

```diff
@@ -0,0 +1,248 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package activity
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/util"
+
+	"github.com/trustbloc/orb/pkg/anchor/activity/generator"
+	"github.com/trustbloc/orb/pkg/anchor/subject"
+)
+
+const (
+	multihashPrefix          = "urn:multihash"
+	multihashPrefixDelimiter = ":"
+
+	anchorEventType    = "AnchorEvent"
+	anchorIndexType    = "AnchorIndex"
+	anchorResourceType = "AnchorResource"
+
+	linkType = "Link"
+
+	idKey             = "id"
+	previousAnchorKey = "previousAnchor"
+	resourceTypeKey   = "type"
+)
+
+// Activity defines anchor activity.
+type Activity struct {
```

Comment:
This isn't really an activity, is it? This object will be embedded within the 'Offer' activity.

Reply:
It is called anchor activity so I put it in activity package under anchor package. Any suggestions for naming?

Reply:
After reading the new spec again, I see that it is called an anchor activity. @troyronda I'm not sure that this is an activity, is it? I believe activities are like actions.

Reply:
Created #611